LICENSE

```diff
@@ -0,0 +1,2 @@
+YEAR: 2023
+COPYRIGHT HOLDER: hmsr authors
```

Comment:
To powinno być tak, że jest i LICENSE, i LICENSE.md?
https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository#including-an-open-source-license-in-your-repository

Reply:
Użyłem funkcji z `devtools` albo podobnego pakietu, która automatycznie dodała te pliki.